package/root/usr/local/sbin/resize_rootfs.sh

```diff
@@ -1,13 +1,24 @@
 #!/bin/bash
 
-set -xe
+if [ $# -eq 0 ]; then
+	echo "No arguments provided... please specify device to resize!"
+	echo "e.g. $(basename $0) /dev/mmcblk1"
+	exit 1
+fi
+
+if [ ! -b "$1" ]; then
+	echo "Specified device '$1' does not exist! Abort!"
+	exit 1
+fi
 
 if [ "$(id -u)" -ne "0" ]; then
 	echo "This script requires root."
 	exit 1
 fi
 
```

Comment:
Should we somehow verify that this device has what we want to rewrite? People can start trying to use that elsewhere.